torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -575,6 +589,7 @@ def kernel_call():
             # reset to zero before evaluating any config
             self.reset_to_zero_args(*args, **kwargs)
             args_with_constexprs = self._get_args_with_constexprs(cloned_args, launcher)
+            validate_args(launcher, *args_with_constexprs, **cloned_kwargs, 2)
```

Comment:
It matters less here than in runtime invocation, but I would still lift this outside of the benchmarking function